trainer/im_utils.py

```diff
@@ -132,7 +133,7 @@ def load_fname(train_annot_dirs, dataset_dir, fname):
 
         # it's possible the image has a different extenstion
         # so use glob to get it
-        fname_no_ext = fname.replace('.nii.gz', '').replace('.nrrd', '')
+        fname_no_ext = fname.replace('.nii', '').replace('.nii.gz', '').replace('.nrrd', '')
```

Comment:
I think this would cause bugs. file_name.nii.gz would be converted to file_name.gz and not file_name 
I think this can be fixed by changing the order of the replacement operations..

Reply:
Good catch! That is of course correct. I will make the edits.

Reply:
I also think that is a good idea. However, I won't have time to look at this for the next two days, as I have other urgent matters. Perhaps you could merge my PR into your own `nii-support` and make the necessary edits? Do you have time?

Reply:
Sure OK.

Reply:
@Abe404 BTW: I wanted to talk to you about something, which preferable should be done in a different platform. Do you have LinkedIn, if so, could you add me and we could talk there?
https://www.linkedin.com/in/andr%C3%A9-pedersen-7166a4162/